gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/tpgm/aggregation/MinTime.java

```diff
@@ -48,9 +42,9 @@ public MinTime(String aggregatePropertyKey, TemporalAttribute interval,
 
   @Override
   public PropertyValue aggregate(PropertyValue aggregate, PropertyValue increment) {
-    if (aggregate.isNull() || aggregate.equals(DEFAULT)) {
-      return increment.equals(DEFAULT) ? PropertyValue.NULL_VALUE : increment;
-    } else if (increment.isNull() || increment.equals(DEFAULT)) {
+    if (aggregate.isNull() || isDefaultValue(aggregate)) {
```

Comment:
This code is now identical to the one in `MaxTime`, except for the call to `PropertyValueUtils.Numeric.min` and `max`, consider unifying those two. Maybe move it to `AbstractTimeAggregateFunction`

Reply:
Done.